### ISSUE TITLE
Audio Background Playback Question

### ISSUE THREAD

**Issue by baehongjun0212** [google/shaka-player#3108]
**What version of Shaka Player are you using?**
"devDependencies": {
    "@teppeis/clutz": "^1.0.29-4c95e12.v20190929",
    "awesomplete": "^1.1.4",
    "babel-core": "^6.26.3",
    "babel-plugin-istanbul": "^4.1.6",
    "babel-polyfill": "^6.26.0",
    "babel-preset-env": "^1.7.0",
    "cajon": "^0.4.4",
    "core-js": "^3.6.5",
    "dialog-polyfill": "^0.5.0",
    "es6-promise-polyfill": "^1.2.0",
    "eslint": "^5.16.0",
    "eslint-config-google": "^0.13.0",
    "eslint-plugin-shaka-rules": "file:./build/eslint-plugin-shaka-rules",
    "esprima": "^4.0.1",
    "fastestsmallesttextencoderdecoder": "^1.0.22",
    "fontfaceonload": "^1.0.2",
    "google-closure-compiler-java": "^20201006.0.0",
    "google-closure-library": "^20200830.0.0",
    "htmlhint": "^0.11.0",
    "jasmine-ajax": "^4.0.0",
    "jimp": "^0.11.0",
    "jsdoc": "github:joeyparrish/jsdoc#45c271fa",
    "karma": "^5.1.1",
    "karma-babel-preprocessor": "^7.0.0",
    "karma-chrome-launcher": "^2.2.0",
    "karma-coverage": "^1.1.2",
    "karma-edge-launcher": "^0.4.2",
    "karma-firefox-launcher": "^1.1.0",
    "karma-ie-launcher": "^1.0.0",
    "karma-jasmine": "^3.3.1",
    "karma-jasmine-ajax": "^0.1.13",
    "karma-opera-launcher": "^1.0.0",
    "karma-safari-launcher": "^1.0.0",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-spec-reporter": "0.0.32",
    "karma-webdriver-launcher": "^1.0.8",
    "less": "^3.13.1",
    "less-plugin-clean-css": "^1.5.1",
    "material-design-lite": "^1.3.0",
    "mux.js": "^5.7.0",
    "pwacompat": "^2.0.10",
    "rimraf": "^2.6.3",
    "sprintf-js": "^1.1.2",
    "stylelint": "^13.8.0",
    "stylelint-config-standard": "^20.0.0",
    "tippy.js": "^4.3.1",
    "ua-parser-js": "github:joeyparrish/ua-parser-js#develop",
    "which": "^1.3.1"
**Please ask your question**

**Is there a way to play the Shakya player only audio without video playback?
Or, I wonder if you have these options or api.
In other words, I'm curious how the content plays only audio without the screen.
Code, api, let me know if you have a link for reference. please...!

These are necessary for work to synchronize.

thank you :)**

**Comment by TheModMaker** [comment]
If the manifest is audio-only, we can play it normally.  If your content normally has video and you want to remove it, you can set the following configuration.

```js
player.configure('manifest.disableVideo', true);
```

**Comment by joeyparrish** [comment]
You can also use an `<audio>` element instead of a `<video>` element to get audio-only playback of any content.

**Comment by baehongjun0212** [comment]
> If the manifest is audio-only, we can play it normally. If your content normally has video and you want to remove it, you can set the following configuration.
> 
> ```js
> player.configure('manifest.disableVideo', true);
> ```

Thank you for your answer. However, this method does not even play audio. I need a way to play audio only and not video :(

**Comment by baehongjun0212** [comment]
> You can also use an `<audio>` element instead of a `<video>` element to get audio-only playback of any content.

Thank you.

**Comment by baehongjun0212** [comment]
Can I play audio only with the Shaka player? The principle of this program is that YouTube provides audio and video streams separately. I wonder how Shaka players can only play audio.

Link: https://github.com/Ashish-Bansal/audio-only-youtube

**Comment by theodab** [comment]
Shaka Player has a configuration value that makes it ignore any video stream:
```js
player.configure('manifest.disableVideo', true);
```
